package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "fastify-response-validation",
+  "version": "0.1.0",
+  "description": "A simple plugin that enables response validation for Fastify.",
+  "main": "index.js",
+  "dependencies": {
+    "ajv": "^6.11.0",
+    "fastify-plugin": "^1.6.0"
+  },
+  "devDependencies": {
+    "ava": "^3.1.0",
```

Comment:
I notice that is like `tap`, with better output: so could we use it in other repo if necessary as an alternative to `tap`?

Reply:
I'm trying it on a few projects. It's very different from `tap`, and very opinionated.
My main concern is that you can't run a test with `node test.js`, but you must use their cli.
Anyhow, we don't force any test framework, as a habit we use `tap`, so feel free to experiment!